ecommerce/extensions/basket/templatetags/basket_tags.py

```diff
@@ -0,0 +1,25 @@
+
+from decimal import Decimal
+from django import template
+from django.conf import settings
+
+register = template.Library()
+
+
+@register.filter(name='calculate_tax')
+def calculate_tax(price_excl_tax):
+    """
+    Calculate the tax that is going to be applied to a product price
+
+    Example:
+        if settings.TAX_RATE is 0.5 and price_excl_tax is 100, this will return 50 (0.5 * 100)
+
+    Arguments:
+        price_excl_tax: Price excluding tax
+    """
+    return Decimal(price_excl_tax) * Decimal(settings.TAX_RATE)
```

Comment:
@alfredchavez, shouldn't we ensure that there are just 2 digits after the decimal place, irrespective of how many digits are there in the tax rate after the decimal point?

Reply:
Sure, I thought as currency does it by its own, it would not be needed, but better to be sure. I've just updated it so it rounds it up to 2 decimal places